src/lookup.cc

```diff
@@ -54,7 +54,7 @@ namespace {
   struct usage_sorter {
     bool operator()(const account_use_pair& left,
                     const account_use_pair& right) const {
-      return left.second > right.second;
+      return left.second < right.second;
```

Comment:
When retro-documenting the --auto-match, I was not able to understand the example in test/baseline/opt-auto-match.test. I then compiled in debug, set the trace, and finally understood that the option --auto-match was buggy. To my understanding, nobody uses the current version as it produces erroneous output. 
While searching documentation on C++ std max_element, I found that the > must be a <. This is what I put as a comment of the commit 80e432a7b269f247e87884f42e49f4d72ca35fe1. To better understand what this --auto-match option is, the reading of test/baseline/opt-auto-match.test should help.